### DISCUSSION TITLE
2ο Παραδοτέο - Π2020132

### DISCUSSION THREAD

**Discussion by Vasilakious** [courses-ionio/sw#1304] [Show and tell]
Για αυτή την βδομάδα, κατέβασα ξανά τη διανομή Arch Linux ως λειτουργικό σύστημα στο laptop μου και εγκατέστησα τα πακέτα [Neofetch](https://github.com/dylanaraps/neofetch), Journalctl, [Asciinema](https://github.com/asciinema/asciinema), Py-Spy και [BashBlog](https://github.com/cfenollosa/bashblog) μέσω του λειτουργικού, χωρίς ιδιαίτερα προβλήματα, αφού είχα ξανακάνει εγκατάσταση του Arch Linux από το μάθημα Επικοινωνία Ανθρώπου και Υπολογιστή σε προηγούμενο εξάμηνο. Τα πακέτα που χρησιμοποίησα είναι χρήσιμα εργαλεία για τη διαχείριση και την παρακολούθηση του συστήματος, και επιτρέπουν την επίλυση προβλημάτων και την αποτελεσματική διαχείριση των εργασιών στο σύστημα, ακόμη και τη συγγραφή δικών μου Blogs και Posts μέσα στο Blog μου.

**Asciinema Neofetch + Journalctl + Py-Spy: https://asciinema.org/a/s5giTRFKSCO106oUrhSLLCOSt**

**Asciinema Blog: https://asciinema.org/a/TjOuAx48ztjZRuOPsmJIzPch0**
(Εδώ δημιουργώ ένα καινούργιο Post στο Blog μου χρησιμοποιώντας κείμενο το οποίο σύνθεσε η τεχνητή νοημοσύνη ChatGPT για την ανάδειξη των ικανοτήτων της στην επικαιρότητα.)
**Αρχείο Blog: [bashblog.tar.gz](https://github.com/courses-ionio/sw/files/10948299/bashblog.tar.gz)**

**[Αναφορά](https://github.com/Vasilakious/sw/tree/2020132/projects/2020132)**
**[Οργανισμός Μου](https://github.com/LetMeDoItForYou)**

**Comment by StavroulaAnatolaki** [comment]
Στοχος αυτου του παραδοτεου ειναι να κανεις μια ασκηση απο τη λιστα. Δεν φτανουν οι εντολες αυτες. Δοκιμασε να ανεβασεις και φωτοφραφιες αυτα που κανεις.

**Reply by aggelos2000430** [reply]
[#1270](https://github.com/courses-ionio/sw/discussions/1270#discussioncomment-5110579) καθώς και [#1305](https://github.com/courses-ionio/sw/discussions/1305)
